### ISSUE TITLE
Screen Reader - Incorrect role assigned inside editor

### ISSUE THREAD

**Issue by JamesNewbyAtMicrosoft** [microsoft/monaco-editor#1485]
**monaco-editor version:** 0.17
**Browser: Chrome/Edge**
**OS:** 

Repro Steps:
1. Open URL https://preview.create.powerapps.com/studio/#  and login with valid credentials.
2. Navigate to left pane present on PowerApps studio homepage.
3. Navigate to 'new component' button under the 'Components'.
4. Navigate to "Insert' menu button and add Label in the component.
5. Navigate to the Formula edit text box flyout.
6. Inspect the Formula edit text box flyout by developer tools.
7. Verify the issue.

Actual Result:
Incorrect "role=presentation" is defined for the Formula edit text box flyout programmatically.

Expected Result:
role= combo box  should be defined for the formula edit text box flyout.

Note:
This issue also exist in Chrome browser.

MAS Reference:
MAS 4.1.2 - Name, Role, Value https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b248054a6-5e68-4771-9e1e-242fb5025730%7d

**Comment by JamesNewbyAtMicrosoft** [comment]
[
![MAS4 1 2_Role (1)](https://user-images.githubusercontent.com/51926290/59626236-39560400-90f0-11e9-8627-df8b33385a59.png)
](url)

**Comment by JamesNewbyAtMicrosoft** [comment]
Basically, I'm one of the software engineers at Microsoft for PowerApps.  We use monaco inside our formula editor and our accessibility team is reporting an a11y bug regarding role assigned.  I personally don't think there is an issue, but the a11y won't let me close the bug out.  Please assist. Thanks

**Comment by alexdima** [comment]
When focus reaches the Monaco Editor, and the cursor is blinking, the real browser focus goes to a very small `<textarea>`. That dom element has the role `textbox`. The rest of the dom nodes that are used to paint the editor have the role `presentation`, which is correct.

![image](https://user-images.githubusercontent.com/5047891/70642651-db8a1b00-1c3f-11ea-8051-eb719c3c1080.png)
